src/components/lib/ConnectWalletDialog.js

```diff
@@ -2,37 +2,36 @@ import React, { Component, useReducer, useState } from 'react'
 import Check from '../svgs/Check'
 import { useWeb3React } from '@web3-react/core'
 import { InjectedConnector } from '@web3-react/injected-connector'
+import { LedgerConnector } from '../../connectors/ledger'
 
-const SUPPORTED_CHAIN_IDS = [
-	// Mainnet
-	1,
-	// Ropsten
-	3,
-	// Rinkeby
-	4,
-	// Dev chains (Ganache, Geth)
-	123, // Low chainId to workaround ledgerjs signing issues.
-	1337,
-	// Keep testnet
-	1101
-]
+const ETH_CHAIN_ID = process.env.ETH_CHAIN_ID || 1337
```

Comment:
How do we want to deal with `chainID` and `wsURL`? Do we want a similar approach to [token dashboard](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/config/config.json) or pass these variables during the build process? @Shadowfiend @mhluongo

Reply:
bump ^^ @Shadowfiend @mhluongo. Also, if we pass them as env vars in the build process, we need to [preface them with `REACT_APP`](https://create-react-app.dev/docs/adding-custom-environment-variables/) to make them available to the build.

Reply:
Unified approach is the right default here. If we want to muck with it later we can figure it out and articulate why.

Reply:
In the dashboard, we have a map `chainId -> wsURL`. To get a `wsURL`, we must first know the `chain/network ID`. We get the `network/chain ID` from contract artifacts. How can we get a `network/chain ID` from `tbtc.js` before initialization `TBTC`? I mean we need `network/chain ID` before initialization `TBTC` to detect which` wsURL` the dapp should use.

Reply:
@shadowfiend ^^ this is what’s blocking

Reply:
This should be part of the build. tbtc.js makes no direct assumptions on the network setup, since it takes in an externally-initialized web3 instance (though indirectly of course the fact that it depends on specific tBTC artifacts currently imposes a limitation on the connectable network).

For now, doing it the way the token dashboard does sounds perfect. Figuring out how to make builds work across networks is an exercise for another moment.